src/transaction/components/TransactionSponsor.tsx

```diff
@@ -30,18 +27,7 @@ export function TransactionSponsor({
 
   return (
     <div className={cn(text.label2, 'flex', className)}>
-      <p className={color.foregroundMuted}>Free gas</p>
-      {!!sponsorName && (
-        <div className="flex">
-          <p className={cn(color.foregroundMuted, 'pl-2')}>•</p>
-          <p className={cn(color.foregroundMuted, 'pl-2')}>
-            Sponsored by{' '}
-            <span className={cn(text.label1, color.primary)}>
-              {sponsorName}
-            </span>
-          </p>
-        </div>
-      )}
+      <p className={color.foregroundMuted}>Zero transaction fee</p>
```

Comment:
We can still have the `Text` handle this content. In case someone wants to customize it.